jetty-core/jetty-fcgi/jetty-fcgi-client/src/main/java/org/eclipse/jetty/fcgi/client/transport/internal/HttpConnectionOverFCGI.java

```diff
@@ -239,7 +239,7 @@ else if (read < 0)
         catch (Exception x)
         {
             if (LOG.isDebugEnabled())
-                LOG.debug("Unable to fill from endpoint {}", endPoint, x);
+                LOG.atDebug().setCause(x).log("Unable to fill from endpoint {}", endPoint);
```

Comment:
Not necessary here, you are in a catch block already.
The Throwable will exist.